index.html

```diff
@@ -24,88 +24,126 @@
-
-</p>
-
-<br>
-<h2>
-<img src="images/Oonagh_Helen_props.JPG">
-<a id="about-us" class="anchor" href="#about-us" aria-hidden="true"><span class="octicon octicon-link"></span></a>About Us</h2>
-
-<p>When Oonagh Mannix travelled to East Africa in 2014, she witnessed first-hand how difficult it was for the local hospital to send messages out to the community. The previous year’s typhoid epidemic had seen a community awareness of only 15%, meaning people continued to drink unsafe water. There needed to be a simple and effective solution to communications and there came the seed for Polorum (formerly ZvakanakaSMS). </p>
-<p> A year on, a lot of hard work and some funding later, our team is based internationally in the UK and Europe. We also work on the ground in Africa, in collaboration with a local NGO and hospital. Our pilot employs five local young people.</p>
-
-<br>
-<h2>
-<a id="contact-us" class="anchor" href="#contact-us" aria-hidden="true"><span class="octicon octicon-link"></span></a>Contact Us</h2>
-<p>
-We partner with NGOs, organisations and tech companies to make sure our system gets delivered to the communities where it’s needed. Our story has been featured in the <a href="http://www.christophersladdin.com/blog/2015/06/the-startup-interview-helen-williams-oonagh-mannix-founders-of-polorum/">Startup Interviews</a>.
-</p>
-<p>
-Interested in working together? Please email us at <a href="mailto:info@polorum.com">info@polorum.com</a> or tweet us at <a href="http://twitter.com/Polorum">@polorum</a>.
-</p>
-      </section>
-    </div>
+<body>
+
+<!--HEADER-->
+<div class="container-fluid head">
+	<div class="row">
+		<img src="images/logo.png" class="center-block img-header">
+	</div>
+</div>
+	
+	<!--NAVBAR-->
+  <nav class="navbar navbar-inverse" data-spy="affix" data-offset-top="197">
+	  <div class="container-fluid">
+		  	<div class="navbar-header">
+				<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#myNavbar">
+					<span class="icon-bar"></span>
+					<span class="icon-bar"></span>
+					<span class="icon-bar"></span>                        
+				</button>
+			</div>
+			<div class="collapse navbar-collapse" id="myNavbar">
+				<ul class="nav navbar-nav navbar-inverse">
+					<li><a href="#what-we-do" data-toggle="collapse" data-target=".nav-collapse.in">What We Do</a></li>
+					<li><a href="#the-system" data-toggle="collapse" data-target=".nav-collapse.in">The System</a></li>
+					<li><a href="#about-us">About Us</a></li>
+					<li><a href="#contact-us">Contact Us</a></li>
+				</ul>
+			</div>
+	 	</div>
+	</nav>	
+	
+	
+	<div class="container-fluid">	
+	<div class="row-fluid">
+	
+		<div class="col-md-6 col-md-offset-3">
+		<div class="container-fluid">
+		<img class="img-in-text img-responsive" src="images/irisInAfrica.jpg"></div>
+		
+		<!--WHAT WE DO-->
+		<a class="anchor" id="what-we-do"></a>
```

Comment:
So, when I click `What We Do` from the top of the page, it scrolls me nicely to the top of the `What We Do` section, but when I click from a point where I've scrolled down the page, it scrolls me back up to the just above the `What We Do` section so that there's a green strip of the header showing